bin/tsc-watch-monorepo.js

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/authentication
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const build = require('../packages/build');
+const debug = require('debug')('build-watch');
+const chokidar = require('chokidar');
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+const path = require('path');
+
+const MONOREPO_DIR = path.resolve(__dirname, '..');
+const BUILD_DIR = path.resolve(MONOREPO_DIR, '.build');
+
+mkdirp.sync(BUILD_DIR);
```

Comment:
`fs.mkdir()` would do the same thing ... this isn't a nested folder. But see comment above about using `fs-extra`